src/data/AudioService.js

```diff
@@ -20,27 +47,34 @@ const AudioService = {
   /**
    * Retrieve audio details by Id number.
    * SSR-called
+   * @param params
```

Comment:
If this requires the `id` should we expand the definition to convey that?

```suggestion
   * @param {object} params
   * @param {string | number} params.id
```